modules/standard/AutoGpu.chpl

```diff
@@ -62,6 +141,32 @@ module AutoGpu {
     compilerError("'@gpu.blockSize' attribute must have exactly one argument: an integral value for the block size");
   }
 
+  /*
+   This attribute requests that the kernel
+   executes each consecutive ``numIters`` iterations of the affected loop
+   sequentially within the same GPU thread. Users must ensure that
+   the arguments to the "blockSize" and "itersPerThread" attributes
```

Comment:
Why are we talking about the positive blockSize requirement here?

Reply:
Because I copied this part from the tech note :D